Feature/NewBookingFromTopMenu.feature

```diff
@@ -0,0 +1,31 @@
+Feature: New booking from the home page top menu
+
+  Scenario Outline: Making A New Booking from Home Page
```

Comment:
change the scenario name to just New booking

Reply:
Done